src/lib.rs

```diff
@@ -239,22 +266,26 @@ impl Manifest {
     /// replaced by the `other`'s; and descriptors will be aggregated to contain
     /// both set of metadata.
     pub fn merge(&mut self, other: &Manifest) -> Result<()> {
-        use std::collections::hash_map::Entry::{Vacant, Occupied};
+        use std::collections::hash_map::Entry::{Occupied, Vacant};
         self.output_map = other.output_map.clone();
         for (desc_bind, desc_ty) in other.desc_map.iter() {
             match self.desc_map.entry(*desc_bind) {
-                Vacant(entry) => { entry.insert(desc_ty.clone()); },
+                Vacant(entry) => {
+                    entry.insert(desc_ty.clone());
+                }
                 Occupied(mut entry) => {
-                    if let DescriptorType::PushConstant(Type::Struct(dst_struct_ty)) = entry.get_mut() {
+                    if let DescriptorType::PushConstant(Type::Struct(dst_struct_ty)) =
+                        entry.get_mut()
+                    {
                         // Merge push constants scattered in different stages.
                         // This match must success.
                         if let DescriptorType::PushConstant(Type::Struct(src_struct_ty)) = desc_ty {
                             dst_struct_ty.merge(&src_struct_ty)?;
                         } else {
                             unreachable!("push constant merging with push constant");
                         }
-                        // It's guaranteed to be interface uniform so we don't
-                        // have to check that.
```

Comment:
I meant to comment on the previous (the `if let `) branch but it seems rustfmt thought it was for the `else` chunk, which is completely wrong. I don't want any automation to break the train of thought.